packages/components/src/Combobox/components/ComboboxOption/ComboboxOption.test.tsx

```diff
@@ -84,11 +111,124 @@ describe("ComboboxOption", () => {
     });
 
     it("should call selectionHandler when clicked", async () => {
-      POM.renderOption("1", "Michael", [], onSelect);
+      POM.renderOption({
+        id: "1",
+        label: "Michael",
+        selected: [],
+        onSelect,
+      });
 
       await userEvent.click(POM.getOption("Michael"));
 
       expect(onSelect).toHaveBeenCalled();
     });
   });
+
+  describe("when supplying a customRender prop", () => {
+    it("renders the custom content", () => {
+      const option: ComboboxOptionProps = {
+        id: "1",
+        label: "Michael",
+        customRender: ({ id, label, isSelected }) => {
+          return (
```

Comment:
What are your thoughts and pulling this custom render out and re-using it across tests? I noticed it is repeated below and the other `customRender` tests can be updated to use this as well